### PR TITLE
[16.0][MIG] fetchmail_thread_default

### PR DESCRIPTION
**Pull request by ntsirintanis**

### CODE REVIEW

File: fetchmail_thread_default/tests/test_fetchmail.py [hunk 1]
```diff
@@ -41,6 +41,7 @@ def test_unbound_incoming_email(self):
         ).message_process(
             self.server.object_id.model,
             MAIL_TEMPLATE.format(
+                return_path="spombot@example.com",
```
Comment (thomaspaulb) [inline fetchmail_thread_default/tests/test_fetchmail.py]: spombot?

**Review by hbrunn — APPROVED** [review]
apart from the typo

**Review by thomaspaulb — APPROVED** [review]

**Comment by ntsirintanis** [comment]
Red because

```
2024-03-19 08:22:47,852 369 ERROR odoo odoo.addons.mail_tracking.tests.test_mail_tracking: FAIL: TestMailTracking.test_db_env_no_cr
Traceback (most recent call last):
  File "/__w/social/social/mail_tracking/tests/test_mail_tracking.py", line 451, in test_db_env_no_cr
    response = controller.mail_tracking_open(db, tracking.id, False)
  File "/opt/odoo/odoo/http.py", line 697, in route_wrapper
    result = endpoint(self, *args, **params_ok)
  File "/__w/social/social/mail_tracking/controllers/main.py", line 56, in mail_tracking_open
    with db_env(db) as env:
  File "/usr/lib/python3.10/contextlib.py", line 135, in __enter__
    return next(self.gen)
  File "/__w/social/social/mail_tracking/controllers/main.py", line 29, in db_env
    yield api.Environment(cr, SUPERUSER_ID, {})
  File "/opt/odoo/odoo/api.py", line 527, in __new__
    assert isinstance(cr, BaseCursor)
AssertionError
```

**Comment by hbrunn** [comment]
please review/merge https://github.com/OCA/social/pull/1332, then rebase

**Review by thomaspaulb — APPROVED** [review]

**Comment by hbrunn** [comment]
/ocabot migrate fetchmail_thread_default (that you can do yourself when you submit a migration)
/ocabot merge nobump

**Pull request merged**